src/main/flight/pid.c

```diff
@@ -771,14 +774,32 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     const float tpaFactor = getThrottlePIDAttenuation();
     const float motorMixRange = getMotorMixRange();
+    const float throttleHpf = getThrottleHpf();
 
 #ifdef USE_YAW_SPIN_RECOVERY
     const bool yawSpinActive = gyroYawSpinDetected();
 #endif
 
     // Dynamic i component,
     // gradually scale back integration when above windup point
-    const float dynCi = MIN((1.0f - motorMixRange) * ITermWindupPointInv, 1.0f) * dT * itermAccelerator;
+    float dynCi;
+#if defined(USE_THROTTLE_BOOST)
+    if (antiGravityNew && (throttleBoost > 0)) {
+    // Calculate new smooth anti-gravity effect where I accumulates faster with faster throttle movement
+    // ThrottleHpf typically reaches max of 0.1 for fast throttle changes
+    // AG Gain of 5000 results in itermAccelerateNew of 5 at throttleHpf of 0.1
+    // if throttle movement is zero, throttleHpf is zero, I accumulates normally
+        float itermAccelerateNew = (1 + (fabsf(throttleHpf) * 0.01 * (pidProfile->itermAcceleratorGain - 1000)));
```

Comment:
There's a double literal `0.01` sneaking into BF code :)

Reply:
oh so it should be 0.01f ?  whoops